ee/codegen/src/__test__/__snapshots__/graph-attribute.test.ts.snap

```diff
@@ -173,3 +173,12 @@ exports[`Workflow > graph > should be correct for two branches merging from sets
 }
 "
 `;
+
+exports[`Workflow > graph > should define nested sets of nodes without compilation errors 1`] = `
+"TopNode >> {
+    OutputTopNode,
+    OutputMiddleNode,
+    OutputBottomNode,
+}
```

Comment:
Old graph was outputting:

```python
TopNode >> {
    {
        OutputTopNode,
        OutputMiddleNode,
    }
    OutputBottomNode,
}
```

Which fails to even compile since sets are unhashable and can't live in a set